sql/maintenance_utils.sql

```diff
@@ -39,11 +39,24 @@ CREATE OR REPLACE FUNCTION @extschema@.compress_chunk(
     hypercore_use_access_method BOOL = NULL
 ) RETURNS REGCLASS AS '@MODULE_PATHNAME@', 'ts_compress_chunk' LANGUAGE C VOLATILE;
 
+-- Alias for compress_chunk above.
+CREATE OR REPLACE PROCEDURE @extschema@.convert_to_columnstore(
+    uncompressed_chunk REGCLASS,
+    if_not_compressed BOOLEAN = true,
+    recompress BOOLEAN = false,
+    hypercore_use_access_method BOOL = NULL
+) AS '@MODULE_PATHNAME@', 'ts_compress_chunk' LANGUAGE C;
+
 CREATE OR REPLACE FUNCTION @extschema@.decompress_chunk(
     uncompressed_chunk REGCLASS,
     if_compressed BOOLEAN = true
 ) RETURNS REGCLASS AS '@MODULE_PATHNAME@', 'ts_decompress_chunk' LANGUAGE C STRICT VOLATILE;
 
+CREATE OR REPLACE PROCEDURE @extschema@.convert_to_rowstore(
+    uncompressed_chunk REGCLASS,
```

Comment:
This name doesn't make any sense, not even before this change. Why would someone give an "uncompressed chunk" as argument to decompress?

```suggestion
    chunk REGCLASS,
```